packages/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -361,6 +361,23 @@ class Transforms {
           }
         }
 
+        // Decrypt encrypted text map if present
+        if (activity.encryptedTextKeyValues !== undefined) {
+          activity.encryptedTextKeyValues.forEach((encryptedData, key) => {
+            promises.push(requestWithRetries(ctx.webex.internal.encryption, ctx.webex.internal.encryption.decryptText,
+              [activity.encryptionKeyUrl, encryptedData])
+              .then((decryptedMessage) => {
+                activity.encryptedTextKeyValues.set(key, decryptedMessage);
+              })
+              .catch((reason) => {
+                ctx.webex.logger.error(`Decrypt activity.encryptedTextKeyValues error for activity ${activity.activityId} in container ${activity.targetId}: ${reason}`);
```

Comment:
Need a unit test to check the behavior of the catch.

Reply:
Sure.